src/test/scala/org/virtuslab/beholder/form/FormFiltersTestForLargeElements.scala

```diff
@@ -0,0 +1,162 @@
+package org.virtuslab.beholder.form
+
+import org.virtuslab.beholder.filters.forms.{ FormFilterComponent, FormFormatterComponent, FromFilterFieldsComponent }
+import org.virtuslab.beholder.filters.{ FilterDefinition, Order }
+import org.virtuslab.beholder.model.{ LargeElement, LargeElementId }
+import org.virtuslab.beholder.{ BaseTest, LargeElementsViewComponent }
+import org.virtuslab.unicorn.{ UnicornPlay, UnicornWrapper }
+
+import scala.concurrent.ExecutionContext.Implicits.global
+
+class FormFiltersTestForLargeElementsRepository(override val unicorn: UnicornPlay[Long])
+  extends LargeElementsViewComponent
+  with FormFormatterComponent
+  with FormFilterComponent
+  with FromFilterFieldsComponent
+  with UnicornWrapper[Long] {
+
+  import FromFilterFields.{ inIntField, inText }
+  import unicorn._
+  import unicorn.profile.api._
+
+  def createFilter = new CustomTypeMappers {
+    val filterGenerator = new FormFilters[LargeElement].create(
+      viewQuery,
+      FromFilterFields.ignore[Option[LargeElementId]],
+      inText,
+      inIntField,
+      inText,
+      inText,
+      inText,
+      inText,
+      inText,
+      inText,
+      inText,
+      inText,
+      inText,
+      inText,
+      inText,
+      inText,
+      inText,
+      inText,
+      inText,
+      inText,
+      inText,
+      inText,
+      inText,
+      inText)
+  }.filterGenerator
+
+}
+
+class FormFiltersTestForLargeElements extends BaseTest {
```

Comment:
maybe test also json filters for large elements?

Reply:
done